deployment/terraform/create.go

```diff
@@ -599,8 +603,10 @@ func (t *Terraform) updateAppConfig(siteURL string, sshc *ssh.Client, jobServerE
 	cfg.SqlSettings.DataSourceReplicas = readerDSN
 	cfg.SqlSettings.MaxIdleConns = model.NewInt(100)
 	cfg.SqlSettings.MaxOpenConns = model.NewInt(100)
+	cfg.SqlSettings.Trace = model.NewBool(false) // Can be enabled for specific tests, but defaulting to false to declutter logs
```

Comment:
It's anyways set to false by default. I guess we are being explicit here?

Reply:
Yes, I like it to beg explicit in case a user wants to change it at some point.